python/apps/taiga/src/taiga/workflows/api/__init__.py

```diff
@@ -145,25 +150,57 @@ async def update_workflow_status(
     """
     Updates a workflow status in the given project workflow
     """
-    workflow = await get_workflow_or_404(project_id=project_id, workflow_slug=workflow_slug)
-    await check_permissions(permissions=UPDATE_WORKFLOW_STATUS, user=request.user, obj=workflow)
-    workflow_status = await get_workflow_status_or_404(workflow_id=workflow.id, workflow_status_slug=slug)
+    workflow_status = await get_workflow_status_or_404(project_id, workflow_slug, slug)
+    await check_permissions(permissions=UPDATE_WORKFLOW_STATUS, user=request.user, obj=workflow_status)
 
     return await workflows_services.update_workflow_status(
-        workflow=workflow,
         workflow_status=workflow_status,
         values=form.dict(exclude_unset=True),
     )
 
 
+################################################
+# delete workflow status
+################################################
+
+
+@routes.stories.delete(
+    "/projects/{project_id}/workflows/{workflow_slug}/statuses/{slug}",
+    name="project.workflowstatus.delete",
+    summary="Delete a workflow status",
+    responses=ERROR_404 | ERROR_403,
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+async def delete_workflow_status(
+    project_id: B64UUID,
+    workflow_slug: str,
+    slug: str,
+    request: AuthRequest,
+    query_params: DeleteWorkflowStatusQuery = Depends(),
+) -> None:
+    """
+    Deletes a workflow status in the given project workflow, providing the option to replace the stories it may contain
+    to any other existing workflow status in the same workflow.
```

Comment:
is  here where we agreed to add the documentation?

Reply:
This change is still missing. I'll include here also the description for the param "moveTo" (removing it from the `Query(Query(description=MOVE_TO_DESC, default=None))` part)

Reply:
I've just uploaded an informal description including the "move_to" query param.
The idea is to update it once we finally find the proper way to document this in FastApi